pkg/csi/cinder/openstack/openstack.go

```diff
@@ -81,16 +83,17 @@ type Config struct {
 }
 
 func logcfg(cfg Config) {
-	klog.V(5).Infof("AuthURL: %s", cfg.Global.AuthUrl)
-	klog.V(5).Infof("Username: %s", cfg.Global.Username)
-	klog.V(5).Infof("UserId: %s", cfg.Global.UserId)
-	klog.V(5).Infof("TenantId: %s", cfg.Global.TenantId)
-	klog.V(5).Infof("TenantName: %s", cfg.Global.TenantName)
-	klog.V(5).Infof("DomainName: %s", cfg.Global.DomainName)
-	klog.V(5).Infof("DomainId: %s", cfg.Global.DomainId)
-	klog.V(5).Infof("TrustID: %s", cfg.Global.TrustID)
-	klog.V(5).Infof("Region: %s", cfg.Global.Region)
-	klog.V(5).Infof("CAFile: %s", cfg.Global.CAFile)
+	klog.Infof("AuthURL: %s", cfg.Global.AuthUrl)
+	klog.Infof("Username: %s", cfg.Global.Username)
+	klog.Infof("UserId: %s", cfg.Global.UserId)
+	klog.Infof("TenantId: %s", cfg.Global.TenantId)
+	klog.Infof("TenantName: %s", cfg.Global.TenantName)
+	klog.Infof("DomainName: %s", cfg.Global.DomainName)
+	klog.Infof("DomainId: %s", cfg.Global.DomainId)
+	klog.Infof("TrustID: %s", cfg.Global.TrustID)
+	klog.Infof("Region: %s", cfg.Global.Region)
+	klog.Infof("CAFile: %s", cfg.Global.CAFile)
+	klog.Infof("Block storage opts: %v", cfg.BlockStorage)
```

Comment:
do we need any doc update for this block storage opts?

Reply:
Its already there